samples/apps/android/LwcTestApp/.gitignore

```diff
@@ -0,0 +1,14 @@
+/*.iml
```

Comment:
I don't think this should start with a `/`—the standard is to ignore *all* .iml files down the project path.

Reply:
🤦‍♂️